lldb/tools/lldb-dap/Handler/LaunchRequestHandler.cpp

```diff
@@ -23,7 +23,7 @@ namespace lldb_dap {
 /// Launch request; value of command field is 'launch'.
 Error LaunchRequestHandler::Run(const LaunchRequestArguments &arguments) const {
   // Validate that we have a well formed launch request.
-  if (!arguments.launchCommands.empty() && arguments.runInTerminal)
+  if (!arguments.launchCommands.empty() && arguments.terminal)
```

Comment:
```suggestion
  if (!arguments.launchCommands.empty() && arguments.terminal != Protocol::eConsole)
```
explicitly check if it not console.